docs/docs/icicle/overview.md

```diff
@@ -13,35 +13,35 @@ title: ICICLE Overview
+- **High-Speed Cryptographic Math:** ICICLE delivers optimized performance for core primitives like elliptic curve operations, MSM, NTT, and Poseidon hash—essential building blocks in modern cryptographic protocols.
 - **Set of Libraries:** ICICLE includes a comprehensive set of libraries supporting various fields, curves, and other cryptographic needs.
 - **Cross-Language Support:** Available bindings for C++, Rust, Go, and potentially Python make ICICLE accessible across different development environments.
-- **Backend Agnosticism:** Develop on CPU and deploy across various backends, including GPUs, specialized hardware, and emerging platforms, based on your project's needs.
-- **Extensibility:** Built for seamless integration and expansion, enabling you to develop and deploy custom backends and cryptographic primitives.
+- **Backend Agnosticism:** Develop on CPU and deploy across various backends, including GPUs, Metal, specialized hardware, and emerging platforms, based on your project's needs.
+- **Extensibility:** Built for seamless integration and expansion, ICICLE enables you to develop and deploy custom backends and cryptographic primitives.
 
 ## Evolution from v2 to v3
 
-Originally, ICICLE was focused solely on GPU acceleration. With the release of v3, ICICLE now supports multiple backends, making it more versatile and adaptable to different hardware environments. Whether you're leveraging the power of GPUs or exploring other compute platforms, ICICLE v3 is designed to fit your needs.
+Originally focused solely on GPU acceleration, ICICLE has evolved. With the release of v3, it now supports multiple backends, making it more versatile and adaptable across diverse hardware environments. Whether you're harnessing the power of GPUs or exploring alternative compute platforms, ICICLE v3 is built to meet your needs.
 
 ## Who Uses ICICLE?
 
 ICICLE has been successfully integrated and used by leading cryptography companies such as [Brevis](https://www.ingonyama.com/blog/icicle-case-study-accelerating-zk-proofs-with-brevis), [Gnark](https://github.com/Consensys/gnark), [Zircuit](https://www.ingonyama.com/blog/case-study-accelerating-zircuits-zero-knowledge-proofs-with-icicle), [zkWASM](https://www.ingonyama.com/blog/how-icicle-helps-grow-the-zkwasm-ecosystem), [Kroma Network](https://www.ingonyama.com/blog/icicle-case-study-accelerating-zk-proofs-with-kroma-network) and others to enhance their ZK proving pipelines.
 
 ## Don't Have Access to a GPU?
 
-We understand that not all developers have access to GPUs, but this shouldn't limit your ability to develop with ICICLE. Here are some ways to gain access to GPUs:
+We understand that not all developers have access to GPUs—but that shouldn’t limit your ability to build with ICICLE. Here are a few ways to get access:
```

Comment:
Not having access to GPUs no longer prevents building with ICICLE since we support CPU and partially Metal.

Maybe change the phrasing here to focus on "experiencing the acceleration" that icicle gives